Dockerfile

```diff
@@ -18,7 +18,7 @@ COPY . .
 RUN CGO_ENABLED=0 GOOS=linux GOARCH=amd64 go build -o ./bin/api
 
 # final stage
-FROM alpine:3.9
+FROM alpine:3.13
 COPY --from=builder /app /app
 
```

Comment:
could it be possible to also create & use a non-root user for the runtime image ?

Reply:
I was under the assumption we were not running with Alpine, seems weird we don't actually run a scratch image. You know the reason @nielstenboom ?

Reply:
No clue, all Davide's work from a long while ago

Reply:
Would it make sense to move to a scratch image in stead of alpine? I think it's overkill to use an OS for Go.

Reply:
make sense to me. A Scratch image is more secured at least (you have no extra deps)

Reply:
sounds good 👍